src/app/container/container.component.ts

```diff
@@ -238,7 +238,7 @@ export class ContainerComponent extends Entry implements AfterViewInit, OnInit {
   }
 
   public setupPublicEntry(url: String) {
-    if (url.includes('/containers/github.com')) {
+    if (url.includes('/containers/github.com') || url.includes('/tools/github.com')) {
```

Comment:
Doesn't this mean an appTool can have either of these urls? Is that right?

Reply:
Yeah, fixing a mistake of mine from: https://github.com/dockstore/dockstore-ui2/pull/1388/files#diff-b2dfc2a868f3f05106c2b4d3f6063d6cd4487782d13085c4083a3b482b2baaa4L227